eap73-rhel8-byos-multivm/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.rhel.jboss.azure</groupId>
+    <artifactId>eap73-rhel8-byos-multivm</artifactId>
+    <version>1.0.0</version>
+    
+    <parent>
+        <groupId>com.microsoft.azure.iaas</groupId>
+        <artifactId>azure-javaee-iaas-parent</artifactId>
+        <version>1.0.13</version>
+        <relativePath></relativePath>
+    </parent>
+    <packaging>pom</packaging>
+    <name>${project.artifactId}</name>
+    <properties>
+        <artifactsLocationBase>https://raw.githubusercontent.com/oracle/${git.repo}/${git.tag}/rhel-jboss-templates</artifactsLocationBase>
+        <template.validation.tests.directory>${basedir}/../../../arm-ttk/arm-ttk</template.validation.tests.directory>
```

Comment:
@edburns `oracle` ?

Reply:
@jamesfalkner Hi James, it is an unused property, I will remove it along with the code change related to making pipeline works with "the new way"